bootstrap.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set +x
+
+# Ask the user for the installation directory
+read -p "Enter the installation directory (e.g., '.' or 'docs'): " install_directory
+
+# Clone the starter pack repository
+echo "Cloning the starter pack repository..."
+git clone git@github.com:canonical/sphinx-docs-starter-pack temp-starter-pack
+rm -rf temp-starter-pack/.git
+
+# Create the specified installation directory
+echo "Creating the installation directory: $install_directory"
+mkdir -p "$install_directory"
```

Comment:
Will that work if the directory is `.`?

Reply:
Yes - at least I tested it and it seems to work.